protos/escrow/escrow.proto

```diff
@@ -78,6 +79,7 @@ message EscrowContract {
   Schedule payout_schedule = 9; // schedule enum
   int32 num_payouts = 10; // numbers of payouts
   bytes reference = 11; // reference like hash value etc
+  int32 customize_paydays = 12; // customized payout days
```

Comment:
is this mean customize_pay_per_days? if it is, suggest change to this name to avoid confusion.

Reply:
it is how many days as payout period, for example payout every 5 days, minimum is 1 day, how about customize_payout_days?